ReactWindows/ReactNative.Shared/UIManager/ViewManager.cs

```diff
@@ -107,6 +108,12 @@ public void UpdateProperties(TFrameworkElement viewToUpdate, ReactStylesDiffMap
         public TFrameworkElement CreateView(ThemedReactContext reactContext)
```

Comment:
>CreateView [](start = 33, length = 10)

Inlines don't use this base class. Will you need to add `AccessibilityHelper.OnViewInstanceCreated` to the `DependencyObjectViewManager` base class?

Reply:
@rozele From the description in the other comment - No. This is for Border, Canvas and Control. Inlines don't need it, so we should be OK if it is invoked here.

---
In reply to: [170249082](https://github.com/Microsoft/react-native-windows/pull/1645#discussion_r170249082) [](ancestors = 170249082)